rusk/src/lib/http/chain.rs

```diff
@@ -61,6 +63,20 @@ impl HandleRequest for RuskNode {
                 self.alive_nodes(amount).await
             }
             (Target::Host(_), "Chain", "info") => self.get_info().await,
+            (Target::Host(_), "Chain", "gas") => {
+                let max_transactions = match request
+                    .event
+                    .data
+                    .as_string()
+                    .trim()
+                    .parse::<usize>()
+                {
+                    Ok(num) if num > 0 => Some(num),
```

Comment:
If user ask for 0 is not our business. We'll give back 0txs

Reply:
Then it would always return 1 for gas prices, instead of the average for the mempool as a whole. I think the current approach would be the expected behavior, giving the actual average.